dremio/Pluralfile

```diff
@@ -0,0 +1,6 @@
+REPO dremio
+ATTRIBUTES CapitalAudience repository.yaml
```

Comment:
This should be `ATTRIBUTES Plural repository.yaml` (unfortunately plural can be the only publisher in this repo)

Reply:
Okay that was fixed @michaeljguarino